docs/release-procedure.md

```diff
@@ -0,0 +1,30 @@
+# Release procedure
+
+## Alert the team
+
+A release PR should not happen while there are remaining work items in QA.
+
+Consult the front-end team if there are still items in QA, or if other previous merges need to be omitted from the release.
+
+The develop branch should be "frozen" during the release procedure; please be sure that everyone on the team is notified of the freeze and when it is completed.
+
+## Merge and create the PR
+
+First, resolve any potential merge conflicts with main on the develop branch
+
+1. `git checkout develop`
+2. `git pull origin develop`
+3. `git checkout main`
+4. `git merge develop` (there should be no merge conflicts)
+5. Update the version in package.json.
+
+Then, issue a pull request for merging the develop branch into the main branch. Await approval and then merge.
```

Comment:
We're not using the "Squash and Merge" method for this are we? If not, I think that we should be explicit here since it is different than the behavior for our other pull requests?

Reply:
I think squash and merge would be fine. Develop would still have the full commit history but `main` would have a much cleaner history